ome_zarr.py

```diff
@@ -219,6 +246,24 @@ def load_ome_zarr(self):
         return (pyramid, {'channel_axis': 1, **metadata})
 
 
+    def load_ome_masks(self):
+        # look for masks in this dir...
+        mask_names = self.get_mask_names()
+        masks = []
+        for name in mask_names:
+            mask_path = os.path.join(self.zarr_path, name)
+            mask_attrs = self.get_json(f'{name}/.zattrs')
+            colors = {}
+            if 'color' in mask_attrs:
+                color_dict = mask_attrs.get('color')
+                colors = {int(k):self.to_rgba(v) for (k, v) in color_dict.items()}
+            data = da.from_zarr(mask_path)
+            # mask data is 5D (t, c, z, y, x) but each layer in napari is 4D (no C)
+            # NB: Assume we want 'first Channel'
+            data = data[:,0,:,:,:]
```

Comment:
Shouldn't this be handled on the napari side, instead of in this library?

Reply:
ah interesting.  There does seem to be a limitation here in napari if you have multiple (image) channels along with a mask for each one.  In the near(ish) future, we'll have layer "groups" that would let you associate each labels layer with a given image layer... but for now, you basically have no choice but to have nC x 2 unassociated layers in napari if you have a mask for every image channel.

So for here, doing `data[:, 0, :, :, :]` will make sure you only have a single labels layer, but will obviously omit some data.  So you could return a list of `[(data[:,n,:,:,:],meta,'labels') for n in range(nChannels)]` if you want to (and, until we provide layer groups, give them names that indicate which image layer they go with?)

lastly, singleton dimensions are handled well in napari, so if nC == 1, you wouldn't need to squeeze it.

Reply:
Without that line, the masks shape is 5D: `(40, 1, 31, 256, 256)` whereas the Image channels have 4D shape `(40, 31, 256, 256)`, so you get a mismatch of dimensions (masks and Image have independent T sliders):

![Screenshot 2020-07-07 at 15 50 13](https://user-images.githubusercontent.com/900055/86799536-f4d63700-c069-11ea-9505-58987c29607a.png)

In 9f7f878 I've split mask channels into layers, as suggested by @tlambert03 - Thanks.